<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">QPushButton{
    background:rgba(247,215,71,255);
    /*border: 1px solid #2361a0; /* Cambiar el color del borde a azul oscuro */
    border-radius: 10px;
    font: 75 12pt &quot;Archivo&quot;;
/* Cambiar el color de fondo a azul oscuro */
    color: rgba(128,96,0,255);
}
/*
QPushButton:hover{

background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(14, 52, 83, 1), stop:1 rgba(14, 52, 83, 1));
color: black;

}
*/
QGroupBox{
background-color: rgb(255, 255, 255); /* Cambiar el color de fondo a azul oscuro */
}

QLineEdit{
background-color: rgb(255,255,255);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
   /*color: #2361a0;*/
	color:rgba(106,105,103,255);
    font: 75 12pt &quot;Archivo&quot;;

}

QLabel{
color: rgba(106,105,103,255);
    font: 75 12pt &quot;Archivo&quot;;
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Categoria</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lnl_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>571</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Precio</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lnl_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>571</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_cerrar">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_guardar">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>190</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(102,56,178,255);
color: rgb(255, 255, 255)</string>
       </property>
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
